### PR TITLE
refactor(gui): gui framework change from solid to react

### PR DESCRIPTION
**Pull request by rwat17**
- changed gui framework to react because it is more common
  and more friendly for newcomers.
- routing moved from src/pages/Pages to src/main, beside that
  only refactoring were made

### CODE REVIEW

**Review by petermetz — COMMENTED** [review]
@rwat17 LGTM with comments: I'd love some documentation why was this a net positive change. I'm guessing you find it easier to develop against react than solid, but still, if there were specific pain points that this addressed it would be good to list them both in the PR description and the **commit message as well**. 

The core idea behind this is that people who come on board to contribute to/maintain the project years from now will get useful context/knowledge from reading the wisdom we leave behind (and have none of that if we don't take time to document our choices and reasoning behind them)

**Comment by rwat17** [comment]
@petermetz Yes, the point of changing is that React is more common among people. It will be easier to develop the package for newcomers too because imo React has that advantage over Solid it is more documented/supported and more people tend to know it.

File: packages/cacti-ledger-browser-react/package.json [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+  ],
+  "scripts": {
+    "build": "yarn run build:prod:frontend",
+    "build:dev:frontend": "vite build --mode=development",
+    "build:prod:frontend": "vite build",
+    "serve": "vite preview",
+    "start": "vite"
+  },
+  "dependencies": {
+    "@emotion/react": "^11.11.3",
```
Comment (outSH) [inline packages/cacti-ledger-browser-react/package.json:56]: pin package versions

Comment (outSH) [reply packages/cacti-ledger-browser-react/package.json:56]: remove vite and react svg

**Review by outSH — CHANGES_REQUESTED** [review]

File: packages/cacti-ledger-browser-react/src/pages/fabric/DashFabric/DashFabric.tsx [hunk 2]
```diff
@@ -0,0 +1,104 @@
+import { supabase } from "../../../supabase-client";
+import CardWrapper from "../../../components/CardWrapper/CardWrapper";
+import { Transaction } from "../../../schema/supabase-types";
+import { Block } from "../../../schema/supabase-types";
+// @ts-expect-error
+import styles from "./DashFabric.module.css";
```
[the remark this answers is not in the document]
Comment (rwat17) [reply packages/cacti-ledger-browser-react/src/pages/fabric/DashFabric/DashFabric.tsx:5]: Removed. They were used in Solid version, idk exactly for what reason, 0 errors after build.

**Review by petermetz — APPROVED** [review]
@rwat17 Thank you!

**Pull request merged**